src/index.ts

```
@@ -1,41 +1,879 @@
+
+    if (typeof args === 'string') {
+      try {
+        parsedArgs = JSON.parse(args)
+        console.error('Parsed string args:', parsedArgs)
+      }
+      catch (e) {
+        console.error('Failed to parse string arguments:', e)
+        // If JSON parsing fails, treat as empty object
+        parsedArgs = {}
+      }
+    } else if (args === null || args === undefined) {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(style/brace-style)